### ISSUE TITLE
adjust parameters in the training 

### ISSUE THREAD

**Issue by Stella1209** [Stella1209/usableml_students#2]
e.g., learning rate, loss function, momentum, dropout-rate

could be applied **before & during** the training

**Comment by lenabaurr** [comment]
TODO: I have to add back-end for choosing loss
